database/migrations/20200530063454-create-assessments.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['use strict' is unnecessary inside of modules.](https://app.codacy.com/gh/BuildForSDG/Farm-Pro/pullRequest?prid=5674718)